lib/v2/screens/profile_screens/guardians/select_guardian/select_guardians_screen.dart

```diff
@@ -18,64 +20,62 @@ class SelectGuardiansScreen extends StatelessWidget {
-                                {
-                                  NavigationService.of(context)
-                                      .navigateTo(Routes.inviteGuardians, state.selectedGuardians),
-                                }
-                            },
+        child: BlocListener<SelectGuardiansBloc, SelectGuardiansState>(
+          listenWhen: (_, current) => current.pageCommand != null,
+          listener: (context, state) {
+            if (state.pageCommand is ShowMaxUserCountSelected) {
+              ScaffoldMessenger.of(context).showSnackBar(SnackBar(
+                  duration: const Duration(seconds: 1),
+                  // ignore: cast_nullable_to_non_nullable
+                  content: Text((state.pageCommand as ShowMaxUserCountSelected).message)));
+            }
+
+            BlocProvider.of<SelectGuardiansBloc>(context).add(ClearPageCommand());
+          },
+          child: BlocBuilder<SelectGuardiansBloc, SelectGuardiansState>(builder: (context, state) {
+            return Scaffold(
+                appBar: AppBar(
+                  title: Text(state.pageTitle, style: Theme.of(context).textTheme.headline7),
+                ),
+                body: Column(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: <Widget>[
+                    Expanded(
+                      flex: 1,
+                      child: Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          Padding(
+                            padding: const EdgeInsets.only(bottom: 4, left: 16, right: 16),
+                            child: state.selectedGuardians.isEmpty
+                                ? const SizedBox.shrink()
+                                : const SelectedGuardiansWidget(),
+                          ),
+                          Expanded(child: SearchUserWidget(resultCallBack: (MemberModel selectedUser) {
+                            BlocProvider.of<SelectGuardiansBloc>(context).add(OnUserSelected(selectedUser));
+                          })),
+                          Padding(
+                            padding: const EdgeInsets.all(16),
+                            child: FlatButtonLong(
+                              title: 'Next',
+                              onPressed: state.selectedGuardians.isNotEmpty
+                                  ? () => {
+                                        if (state.selectedGuardians.isNotEmpty)
+                                          {
+                                            NavigationService.of(context)
+                                                .navigateTo(Routes.inviteGuardians, state.selectedGuardians),
+                                          }
+                                      }
+                                  : null,
```

Comment:
Why do you check again within the function if `state.selectedGuardians.isNotEmpty `can change?

Reply:
no, thats a mistake